app/views/questionnaire.py

```diff
@@ -37,7 +37,7 @@
 
 logger = get_logger()
 
-questionnaire_blueprint = Blueprint(name='questionnaire',
+questionnaire_blueprint = Blueprint(name='Question',
```

Comment:
This shouldn't have changed name

Reply:
with this change, should be good to go !